core/cibox-project-builder/files/drupal8/scripts/devops/reinstall/profile_workflow.yml

```diff
@@ -4,6 +4,18 @@
   file: name={{ settings_file_path }} state=absent
   sudo: yes
 
+- name: Check php-cli version
+  shell: php -v
+  register: installed_php_cli_version
+
+- name: Start the MySQL service
```

Comment:
This should live in prepare environment playbook, since MySQL is used regardless or workflow(profile or SQL).